gist/README.md

```diff
@@ -0,0 +1,180 @@
+# gist - Manage your gist like a pro
```

Comment:
can you add a link to this readme under the main readme? If you look under the all tools section you should add an entry for gist and put the link to this readme.

Reply:
fixed in d6d1343